app/templates/components/user-list.hbs

```diff
@@ -1,17 +1,25 @@
 <table>
   <thead>
     <tr>
-      <th class='text-left' colspan=1>{{t 'user.firstName'}}</th>
-      <th class='text-left' colspan=1>{{t 'user.lastName'}}</th>
-      <th class='text-left' colspan=1>{{t 'general.school'}}</th>
+      <th class='text-left' colspan=1></th>
+      <th class='text-left' colspan=3>{{t 'user.fullName'}}</th>
+      <th class='text-left' colspan=2>Campus ID</th>
```

Comment:
He's wrong event if he is right.  Al strings need to have a translation.  Event it ends up being the same phrase in all languages.